src/app/homepage/home-page.component.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
```

Comment:
You shouldn't put the html, body and meta tags inside components since they belong in index.html. You should move the style imports there . On another note, using w3schools styles doesn't seem like a good idea, better stick to something that is stable - like bootstrap or angular material